src/channel/rpc_0_7_1.ts

```diff
@@ -60,6 +66,9 @@ export class RpcChannel {
 
   private chainId?: StarknetChainId;
 
+  /**
+   * Connected RPC node specification version
```

Comment:
```suggestion
   * RPC specification version of the connected node
```
This is a suggestion to sync the typedoc note with `this. channelSpecVersion `.